translations/fr.json

```diff
@@ -0,0 +1,224 @@
+        "Chance": "Chance",
+        "Changes provided by": "Changements apportés par",
+        "[Click here]({{url}}) for full notes.": "[Cliquez ici]({{url}}) pour les notes complètes.",
+        "Click the link above to invite the Stash bot to your Discord server!": "Cliquez sur le lien au-dessus pour inviter Stash Bot sur votre serveur Discord !",
+        "Come visit us in our server.": "Venez nous voir sur notre serveur.",
+        "Contributors": "Contributeurs",
+        "Craft": "Fabriquer",
+        "Cures": "Remèdes",
+        "Default progress": "IDK",
+        "Delay": "Délai",
+        "Dmg": "IDK",
+        "Duration": "IDK",
+        "EFT Discord Bot": "Bot Discord d'EFT",
+        "Enjoy": "Amusez-vous bien",
+        "Escape from Tarkov Status": "Statut d'Escape from Tarkov",
+        "Escort": "Escorter",
+        "Examples": "Exemples",
+        "Fee": "Frais",
+        "Flea Market": "Marché aux puces",
+        "Flea Price (avg)": "IDK",
+        "Flea Price (low)": "IDK",
+        "Flea Price (optimal)": "IDK",
+        "Found no results for \"{{searchString}}\"": "Pas de résultats pour la recherche \"{{searchString}}\"",
+        "for": "pour",
+        "Format HH:MM:SS": "Format HH:MM:SS",
+        "Frag": "IDK",
+        "Get the full data from {{url}}": "Obtenez les données complètes à partir de {{url}}",
+        "Good Luck!": "Bonne Chance !",
+        "Health": "Vie",
+        "Help for /": "IDK",
+        "Hideout": "Refuge",
+        "Hideout Management skill set to {{managementLevel}}.": "IDK",
+        "I have been online for": "Je suis en ligne depuis",
+        "Intelligence Center": "IDK",
+        "Invalid token": "IDK",
+        "Item Tier": "Rang d'Objet",
+        "kg": "kg",
+        "Latest EFT Changes": "Dernières modifications d'EFT",
+        "Last Updated": "Dernière mise à jour",
+        "Level": "Niveau",
+        "level": "niveau",
+        "LL": "IDK",
+        "Loot Tiers": "Rang de butin",
+        "Loot tiers are divided primarily by the per-slot value of the item": "IDK",
+        "Map": "Carte",
+        "Map made by {{author}}": "Carte réalisée par {{author}}",
+        "minutes": "minutes",
+        "more": "plus",
+        "Name": "Nom",
+        "Need Help or Have Questions?": "Besoin d'aide ou des questions?",
+        "never": "jamais",
+        "Next update": "Prochaine mise à jour",
+        "No matching hideout station found.": "Aucune station de refuge correspondante n'a été trouvée.",
+        "No matching trader found.": "Aucun commerçant correspondant n'a été trouvé.",
+        "No prices available.": "Pas de prix disponibles..",
+        "Note: Progress synced via [TarkovTracker](https://tarkovtracker.io/settings/) will overwrite your hideout settings. \nUse `/progress unlink` to stop syncing from TarkovTracker.": 
+        "IDK",
+        "Net": "IDK",
+        "Pen": "IDK",
```

Comment:
Pen as in "penetration" in the context of a type of ammunition (bullet) penetrating an armor vest